pycode/memilio-epidata/memilio/epidata/README.rst

```diff
@@ -133,154 +128,85 @@ The data is written either in json or hdf5 format
-        - ## Header
-        - # @brief name descr
-        - # longer description
-    - add description in the beginning of every function directly after the definiton
-        - start and end with """
-        - add a short description to first line
-        - afterwards add a longer description
-        - # @param name of parameter
-        - # @return type description
-
-When you add a new script
-
-- add a executable to the setup.py in "epidemiology/pycode/"
-- add it to the cli_dict in getDataIntoPandasDataFrame.py
-    - add a meaningfull key for the new script
-    - as the value add a list in the form [comment to print when script is started, list of used parser arguments (optional)]
-    - if more than the default parser should be added, add these parser to the  list of used parser
-- add tests
-- add an entry "executablename -h" to the .gitlab-ci.yml
-- add it to getAll.py
-- add generated data to cleanData
-
-Adding a new parser:
-
-- add default value to defaultDict in defaultDict.py
-- add to cli_dict in getDataIntoPandasDataFrame.py which scripts use this parser
-- add an if 'new parser' in what_list and add parser.add_argument()
-
-General
-- Always add unittests
-- Check test coverage report, if every new feature is covered.
-- Check the pylint report just comments with "refactor" are allowed.
-
-More detailed information can be found in the documentation of the different functions in
-
-Some more notes
----------------
-
-When speaking about infected, means always infected inclusive the already recovered persons
-
-There are different columns of infected:
-
-'Confirmed_PCR' means that these infected people were tested and confirmed to be infected by a PCR test
-'Confirmed_AB' means that these infected people were tested and confirmed to be infected by an ANTIBODY test
-'Confirmed_total' is the sum of the previous two
-'Confirmed' if the differentiation between PCR and ANTIBODY is not made/known, only the column 'Confirmed' appears
-
-
-For DIVI:
-
-For everyday there is one file, from which we extract the date.
-However, in the beginning the data was different to the later ones.
-For the first two dates, 24.4. and 25.4., there is no data for ICU_ventilated (faelle_covid_aktuell_beatmet).
-For the 24.4. even has the ICU data only for each state (faelle_covid_aktuell_im_bundesland) but not for every county.
-Thus, it is not yet considered in the summarized data for counties, states and whole Germany. (There are
-zero entries for these dates).
-Not every hospital is reporting the number of corona patients in intensive care units (ICU). The number of
-reporting hospitals differs from day to day and is given in FullData_DIVI.
+If a new functionality shall be added please stick to the instructions in `epidata README <../../README.rst>`_ of the above folder.
+
+For information about testing, coverage, pylint and tools see also the `epidata README <../../README.rst>`_ of the above folder.
```

Comment:
In General: Looks like this is a good seperation from general epidata. Great job seperating each topic. Looks way more concise.